src/DateRangePicker.jsx

```diff
@@ -131,6 +132,18 @@ const DateRangePicker = React.createClass({
     };
   },
 
+  componentWillUpdate(nextProps, nextState){
+    if(nextState.month !== this.state.month || 
+        nextState.year !== this.state.year){
+      if(typeof this.props.onChange === "function"){
+        this.props.onChange({
+          year: nextState.year,
```

Comment:
I wonder if a more specific event should be added here instead of giving back year + month?
Perhaps `onMonthChange` and `onYearChange` would make sense here?